subtrack.MAUI/Services/SubscriptionsCalculator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using subtrack.DAL.Entities;
+
+namespace subtrack.MAUI.Services;
+
+public static class SubscriptionsCalculator
+{
+    public static decimal GetMonthlyCost(IEnumerable<Subscription> subscriptions)
```

Comment:
I'd prefer if it would be an extension method instead, so you can just call `subscriptions.GetMonthlyCost()`

Reply:
Changed the method to an extension method.